### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/webkit2gtk-rs/issues/48

WIP as need include and use last gir, also contain [patch] section in Cargo.toml

### CODE REVIEW

**Comment by antoyo** [comment]
Looks good to me.

**Comment by EPashkin** [comment]
Regeneration with latest gir:
all *-ffi replaces to *-sys.
`web_kit2_sys` IMHO looks slightly ugly, but currently I don't plan add predefined names to gir.
Also added `make regen_check`, so nightly will fail until right submodules added.

**Comment by EPashkin** [comment]
This PR is ready

File: src/auto/authentication_request.rs [hunk 1]
```diff
@@ -18,18 +17,19 @@ use glib::signal::SignalHandlerId;
...
 use glib::translate::*;
 #[cfg(any(feature = "v2_2", feature = "dox"))]
-use glib_ffi;
+use glib_sys;
 #[cfg(any(feature = "v2_2", feature = "dox"))]
 use std::boxed::Box as Box_;
 use std::fmt;
 #[cfg(any(feature = "v2_2", feature = "dox"))]
 use std::mem::transmute;
+use web_kit2_sys;
```
Comment (antoyo) [inline src/auto/authentication_request.rs]: Could you please configure it to generate the name `webkit2_sys` instead?

Comment (EPashkin) [reply src/auto/authentication_request.rs]: You meant add hardcoded replacement to gir?
Or I totally forgot about way to "configure" it.

Comment (EPashkin) [reply src/auto/authentication_request.rs]: Added option to gir https://github.com/gtk-rs/gir/pull/745, will update this PR soon

**Comment by EPashkin** [comment]
Updated, and CI passed

**Comment by antoyo** [comment]
I'm all good with this PR, thank you.
Can I merge it?

**Comment by EPashkin** [comment]
Yes

**Pull request merged**